check_manifest.py

```diff
@@ -406,33 +406,14 @@ def detect(cls, location):
 
     def get_versioned_files(self):
         """List all files versioned by git in the current directory."""
-        files = self._git_ls_files()
-        submodules = self._list_submodules()
-        for subdir in submodules:
-            subdir = os.path.relpath(subdir).replace(os.path.sep, '/')
-            files += add_prefix_to_each(subdir, self._git_ls_files(subdir))
```

Comment:
After this, `add_prefix_to_each()` is no longer used and can be removed as well.

Reply:
Removed in 16f438c :)

Reply:
...or rather db14819, `flake8` was being picky.

Reply:
Thanks!